plugin-desktopswitch/desktopswitch.cpp

```diff
@@ -53,7 +53,7 @@ DesktopSwitch::DesktopSwitch(const ILXQtPanelPluginStartupInfo &startupInfo) :
     mLabelType(static_cast<DesktopSwitchButton::LabelType>(-1))
 {
     m_buttons = new QButtonGroup(this);
-    connect (m_pSignalMapper, SIGNAL(mapped(int)), this, SLOT(setDesktop(int)));
+    connect (m_pSignalMapper, QOverload<int>::of(&QSignalMapper::mapped), this, &DesktopSwitch::setDesktop);
```

Comment:
Would you please update this by using `#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))` and `QSignalMapper::mappedInt` for Qt ≥ 5.15?

Reply:
Sure!
I was waiting for all of these as the minimum Qt dep would be raised to 5.15 for the whole lxqt.... btw, any plans for it? After 0.17 release?

Reply:
For now, we check the version. I think Qt < Qt 5.15 couldn't be supported *after* LXQt 0.17. Qt6 might also be ready for real use (in contrast to testing) until then.